src/lib/constants.ts

```diff
@@ -0,0 +1,7 @@
+const docSiteUrl = 'https://stardust-ui.github.io/react'
+const quickStartUrl = `${docSiteUrl}/quick-start`
```

Comment:
Let's not dupe constants that already exist in the repo.  Let's pull these from `/docs` into `/src`.  The docs are allowed to import src files.

Reply:
done, pulled all constants together